cosin/src/main/java/com/nikitagordia/cosin/Cosin.java

```diff
@@ -1,56 +1,75 @@
-package com.nikitagordia.cosin;
+/*
+ * Copyright (C) 2020-21 Application Library Engineering Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 
-import android.content.Context;
-import android.graphics.Canvas;
-import android.graphics.Color;
-import android.graphics.Paint;
-import android.graphics.Typeface;
-import android.util.AttributeSet;
-import android.view.View;
+package com.nikitagordia.cosin;
 
-import com.nikitagordia.cosin.colorAdapters.DefaultColorAdapterGB;
+import ohos.agp.components.AttrSet;
+import ohos.agp.components.Component;
+import ohos.agp.render.BlendMode;
+import ohos.agp.render.Canvas;
+import ohos.agp.render.Paint;
+import ohos.agp.text.Font;
+import ohos.agp.utils.Color;
+import ohos.app.Context;
+import com.nikitagordia.cosin.colorAdapters.DefaultColoradapterGB;
 import com.nikitagordia.cosin.textAdapters.DefaultBinaryTextAdapter;
 
-
 /**
- * Created by nikitagordia on 04.03.18.
+ * Created by nikitagordia on 05.03.18.
```

Comment:
Please retain the same java doc as in base library

Reply:
Done